controllers/clusterresources/clusterbackup_controller.go

```diff
@@ -131,7 +136,7 @@ func (r *ClusterBackupReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	iBackupEvents := iBackup.GetBackupEvents(backup.Spec.ClusterKind)
 
 	if len(iBackupEvents) < len(backupsList.Items) {
-		err = r.API.TriggerClusterBackup(instaclustr.ClustersEndpointV1, backup.Spec.ClusterID)
+		err = r.API.TriggerClusterBackup(instaclustr.TriggerClusterBackupEndpointV2, backup.Spec.ClusterID, instaclustr.ClusterKindsMap[backup.Spec.ClusterKind])
```

Comment:
Form the URL inside the `TriggerClusterBackup` method please

Reply:
done